system/lib/libc/emscripten_syscall_stubs.c

```diff
@@ -28,7 +28,7 @@ static int g_pid = 42;
 static int g_pgid = 42;
 static int g_ppid = 1;
 static int g_sid = 42;
-static mode_t g_umask = S_IRWXU | S_IRWXG | S_IRWXO;
```

Comment:
Doing a little googling it seems that most linux distros default to 022.  I wonder if we should do that same?

Also, I'm surprised to see now tests failing.. do our filesystems not honor this, or are there just no tests for this?

Reply:
Several tests in CPython test suite are failing on a new buildbot. The same tests are not failing in containerized test environment. The buildbot uses an user account with fewer privileges while the container runs the tests as root with DAC override capability.

The failing code applies the umask directly to permission bits, for example https://github.com/python/cpython/blob/476d30250811e185615dfb971c6a810cac2093bd/Lib/dbm/dumb.py#L306-L317

Reply:
I don't doubt the fix, I'm just surprised we don't have any tests that depend on this in emscripten itself (we should probably add one).